node/agent.go

```diff
@@ -825,3 +872,44 @@ func waitForSsNodes(processing map[string]chan int, ssResultChan chan stateResul
 	glog.V(0).Info("All service state nodes are shut down")
 	return
 }
+
+func (a *HostAgent) createContainer(conf *dockerclient.Config, hostConf *dockerclient.HostConfig, svcID string, instanceID int) (*docker.Container, error) {
```

Comment:
Don't call it createContainer, because we already have a CreateContainer and that doesn't directly call this